dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,22 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 1)]
+    runtimeclass AppNotificationDevicesData
+    {
+        AppNotificationDevicesData();
+
+        static Boolean IsVideoCallingSupported();
+
+        String CameraDeviceId{ get; set; };
+        String MicrophoneDeviceId{ get; set; };
+        String SpeakerDeviceId{ get; set; };
```

Comment:
Add a comment on each property what is means or does similar to AppNotificationProgressData

Reply:
done